zgw/zaken-api/src/main/kotlin/nl/nlportal/zakenapi/client/ZaakDocumentenConfig.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.nlportal.zakenapi.client
+
+import nl.nlportal.documentenapi.domain.Vertrouwelijkheid
+import nl.nlportal.documentenapi.domain.Vertrouwelijkheid.BEPERKT_OPENBAAR
+import nl.nlportal.documentenapi.domain.Vertrouwelijkheid.INTERN
+import nl.nlportal.documentenapi.domain.Vertrouwelijkheid.OPENBAAR
+import nl.nlportal.documentenapi.domain.Vertrouwelijkheid.ZAAKVERTROUWELIJK
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties(prefix = "nl-portal.zgw.zaken-api.zaak-documenten")
+data class ZaakDocumentenConfig(
+    var vertrouwelijkheidsaanduidingWhitelist: List<Vertrouwelijkheid> =
+        listOf(
+            OPENBAAR,
+            BEPERKT_OPENBAAR,
+            INTERN,
+            ZAAKVERTROUWELIJK,
+        ),
+)
```

Comment:
Zou op zijn minst niet de huidige document filters ook aanwezig moeten zijn in de mogelijkheden om op te filteren, dus DocumentStatus.DEFINITIEF, DocumentStatus.GEARCHIVEERD???

Reply:
Heya,
That's not what my story said, but I'll add it anyway since it makes sense to also be able to configure that. Currently   the statuses are fitlered by those two hardcoded values indeed.
Thanks for the feedback.